### PR TITLE
LAM-167 Styling for create lambda instance and lambda instance details

### PR DESCRIPTION
**Pull request by gouzouni625**
# Description

Added styling on web pages for create lambda instance and lambda instance details.

Added error messages when required information for creating a lambda instance is omitted.
Added information alert when a request is being made for creating a new lambda instance.

After the request for creating a new lambda instance has been accepted the user is redirected to the details of the new lambda instance.

### CODE REVIEW

**Comment by gouzouni625** [comment]
Please review @stevelaskaridis @makistsantekidis .

File: webapp/frontend/app/templates/lambda-instance.hbs [hunk 1]
```diff
@@ -1,48 +1,166 @@
...
-    <li>Instance status details: {{model.instance.status_detail}}</li>
-  {{#if lambda-instance.status_failure_message}}
-      <li>Instance failure message: {{lambda-instance.status_failure_message}}</li>
-  {{/if}}
-</ul>
+<!-- Content Header (Page header) -->
+<section class="content-header">
+  <h1>
+    Dashboard
+    <small>Control panel</small>
```
Comment (ioantsaf) [inline webapp/frontend/app/templates/lambda-instance.hbs]: please change this to `<small>Lambda Instance Details</small>`

File: webapp/frontend/app/templates/lambda-instance.hbs [hunk 2]
```diff
@@ -1,48 +1,166 @@
...
-    No apps are currently deployed on the instance
-{{/if}}
-
-{{outlet}}
+          <div class="row">
+            <div class="col-ms-6 col-xs-5">
+              <table  class="table table-bordered table-hover table-responsive">
+                <thead>
+                  <tr>
+                    <th>Name</th>
```
Comment (ioantsaf) [inline webapp/frontend/app/templates/lambda-instance.hbs]: please change this to `<th>Property</th>`

File: webapp/frontend/app/templates/components/create-lambda-instance-form.hbs [hunk 3]
```diff
@@ -1,124 +1,245 @@
...
+          <div class="box-tools pull-right">
+            <button class="btn btn-box-tool" data-widget="collapse"><i class="fa fa-minus"></i></button>
+            <button class="btn btn-box-tool" data-widget="remove"><i class="fa fa-times"></i></button>
+          </div>
+        </div><!-- /.box-header -->
+        <div class="box-body no-padding">
+          <div class="row">
+            <div class="col-md-9 col-sm-8">
+              <div class="pad">
+                Here you can create your own lambda Instance
```
Comment (ioantsaf) [inline webapp/frontend/app/templates/components/create-lambda-instance-form.hbs]: maybe you could change 'lambda' to 'Lambda'

**Comment by ioantsaf** [comment]
+1

**Pull request merged**